db/seeds.rb

```diff
@@ -51,7 +51,7 @@
         service = Service.create!({
             name: Faker::Company.name,
             organisation: org,
-            description: Faker::Lorem.paragraphs(number: 1),
+            description: Faker::Lorem.paragraphs(number: 1).join(' '),
```

Comment:
Since we're only using 1 paragraph, we can do even better and use [Faker::Lorem.paragraph](https://rubydoc.info/github/faker-ruby/faker/master/Faker%2FLorem%2Eparagraph) and avoid needing to do a join!

Reply:
TIL!